docs/source/support/release-notes.rst

```diff
@@ -9,6 +9,37 @@ This page contains release notes for the SDK.
 HEAD — ongoing
 --------------
 
+Ledger API
```

Comment:
@bethaitman: I thought about starting a new trend: component sections in release-notes. WDYT?

Reply:
Let's try it!